Gulpfile.js

```diff
@@ -22,7 +22,15 @@ gulp.task("build", function () {
     }))
     .pipe(through.obj(function (file, enc, callback) {
       file._path = file.path;
-      file.path = file.path.replace(/^([^\\]+)\/src/, "$1/lib");
+      // Replace src with lib
+      var sub = file.path.substr(__dirname.length);
```

Comment:
It started out as a fix to deal with backslashes. Then, I mistakenly assumed that it would fail on paths that contain `src` outside babel's root. I guess you could also run it through `slash` to always get forward slashes, although my approach should still be safer in that it won't fail if the path to the file contains multiple instances of `src` inside babel's root.